escavenge.py

```diff
@@ -0,0 +1,44 @@
+import database
+import gamemaker
+import team
+
+class Escavenge():
+
+  def __init__(self):
+    self.database = database.Database()
+    self.game_maker = gamemaker.GameMaker(database)
+    self.team = team.Team(database)
+    maker = {}
+    user = {}
+
+    maker["logout"] = lambda: self.game_maker.logout()
+    user["logout"] = lambda: self.team.logout()
+
+  def login(self,username, password):
+      if(username is "maker"):
+          if(self.game_maker.login(username, password))):
+              print("Success! Logged in as ")
+                return True
+      else:
```

Comment:
You'l have to get the team that matches the username and password, then use its login function